### PR TITLE
1.11.0.0?

### PR DESCRIPTION
**Pull request by Frosty-J**
Dunk dunk dunk dunk dunk!

Obviously ensure things are done to the Liftoff standards, because I don't know what I'm doing - even in regards to versions, let alone everything else. Not fully tested but libGDX 1.11.0 launches fine.

### CODE REVIEW

File: src/main/kotlin/gdx/liftoff/data/libraries/unofficial/thirdPartyExtensions.kt [hunk 1]
```diff
@@ -715,7 +715,7 @@ class RegExodus : ThirdPartyExtension() {
  * UI toolkit with extra widgets and a different theme style.
  * Check the vis-ui changelog for what vis-ui versions are compatible
  * with which libGDX versions; vis-ui 1.5.0 is the default and is
- * compatible with libGDX 1.10.0.
+ * compatible with libGDX 1.11.0.
```
Comment (tommyettinger) [inline src/main/kotlin/gdx/liftoff/data/libraries/unofficial/thirdPartyExtensions.kt:718]: Are we sure vis-ui 1.5.0 is actually compatible with 1.11.0 ? Last time this required telling vis-ui to skip a version check.

File: build.gradle [hunk 2]
```diff
@@ -26,7 +26,7 @@ buildscript {
     }
 }
 
-version = "1.10.0.15-SNAPSHOT"
+version = "1.11.0.1-SNAPSHOT"
```
Comment (tommyettinger) [inline build.gradle:29]: This version is tricky and probably has to be 1.11.0.0 for a release, though there will be one commit with this 1-SNAPSHOT version earlier in time than the 1.11.0.0 release... huh...

**Review by tommyettinger — APPROVED** [review]
You beat me to the punch, but I already had some of these steps in. I'll commit my changes, which partially overlap yours and partially extend, then merge yours even with conflicts so I can properly go through the parts that I want to adjust (mostly versioning).

File: src/main/kotlin/gdx/liftoff/data/libraries/unofficial/thirdPartyExtensions.kt (hunk 1 above)
Comment (Frosty-J) [reply src/main/kotlin/gdx/liftoff/data/libraries/unofficial/thirdPartyExtensions.kt:718]: [https://github.com/kotcrab/vis-ui/blob/9706f77/ui/src/main/java/com/kotcrab/vis/ui/VisUI.java#L96-L103](https://github.com/kotcrab/vis-ui/blob/9706f7764749c14bec18797d06f66f76cf28261a/ui/src/main/java/com/kotcrab/vis/ui/VisUI.java#L96-L103) fires but the demo runs fine once `sourceCompatibility` is updated. Is vis-ui really this sensitive to version changes?

**Pull request merged**